plugin.xml

```diff
@@ -150,16 +150,16 @@
 				<source url="https://cdn.cocoapods.org/"/>
 			</config>
 			<pods use-frameworks="true">
-				<pod name="Firebase/Core" spec="10.10.0"/>
-				<pod name="Firebase/Auth" spec="10.10.0"/>
-				<pod name="Firebase/Messaging" spec="10.10.0"/>
-				<pod name="Firebase/Performance" spec="10.10.0"/>
-				<pod name="Firebase/RemoteConfig" spec="10.10.0"/>
-				<pod name="Firebase/InAppMessaging" spec="10.10.0"/>
-				<pod name="Firebase/Firestore" spec="10.10.0"/>
-				<pod name="Firebase/Crashlytics" spec="10.10.0"/>
-				<pod name="Firebase/Functions" spec="10.10.0"/>
-				<pod name="Firebase/Installations" spec="10.10.0"/>
+				<pod name="FirebaseCore" spec="10.10.0"/>
+				<pod name="FirebaseAuth" spec="10.10.0"/>
+				<pod name="FirebaseMessaging" spec="10.10.0"/>
+				<pod name="FirebasePerformance" spec="10.10.0"/>
+				<pod name="FirebaseRemoteConfig" spec="10.10.0"/>
+				<pod name="FirebaseInAppMessaging" spec="10.10.0-beta"/>
```

Comment:
Thanks for this PR.
Can I ask why this is set to use the beta version of InAppMessaging?

Reply:
For some reason it seems like there are only `beta` versions in CocoaPods - that could be a misunderstanding on my part, but I could not find a non-beta build

Reply:
Okay, fair enough. The pinned version can be overridden at install time anyway so I think that's fine.